src/System/Wlog/LoggerNameBox.hs

```diff
@@ -39,52 +39,52 @@ import           System.Wlog.LoggerName      (LoggerName)
 -- TODO: replace current methods with Lens?
 class HasLoggerName m where
     -- | Extract logger name from context
-    getLoggerName :: m LoggerName
+    askLoggerName :: m LoggerName
```

Comment:
Could you please add `default` implementations of `askLoggerName` and `modifyLoggerName` (using `hoist` from `MFunctor`? This will help to remove boilerplate.

Just see how it's done here for `CanLog` type class:

https://github.com/serokell/log-warper/blob/ec162c6557ec25939c7fa37bfaaa3c4a0682ba5a/src/System/Wlog/CanLog.hs#L70